### DISCUSSION TITLE
无法正确显示歌词

### DISCUSSION THREAD

**Discussion by glass-walls** [amll-dev/applemusic-like-lyrics#283] [Q&A] [answered]
![1730548073767](https://github.com/user-attachments/assets/ab421602-151a-4f54-98b3-5f175129beb4)
现在使用的时候都不显示歌词了，只有一个封面，多次重载也没用

**Comment by sorahane0227** [comment]
同，歌词闪一下就消失，移除了所有其他插件也不行

**Reply by Hamilton-Liu** [reply]
在歌词源选项里把网易云歌词源置顶即可解决（作者不知道什么时候偷偷更新了这个选项

**Comment by WonderfulStone** [comment]
同问，粘贴歌词点保存后歌词消失，播放也无歌词显示

https://github.com/user-attachments/assets/91e0366b-759c-49c4-a53b-e30abf7d3cd5

**Comment by WonderfulStone** [comment]
更新到最新版本可解决

**Comment by Hamilton-Liu** [comment, marked as answer]
在歌词源选项里把网易云歌词源置顶
